scala/scala-impl/test/org/jetbrains/plugins/scala/lang/macros/ScalazDerivingTest.scala

```diff
@@ -126,17 +126,17 @@ case object <caret>Caz
 
 }
 
-object StalactiteTest {
+object ScalazDerivingTest {
 
-  case class StalactiteLoader() extends IvyLibraryLoaderAdapter {
+  case class ScalazDerivingLoader() extends IvyLibraryLoaderAdapter {
```

Comment:
we can now load test dependencies directly instead of using the testDownloader sbt config. eg in ShapelessTest:
```
override protected def loadIvyDependencies(): Unit =
    DependencyManager("com.chuusai" %% "shapeless" % "2.3.2").loadAll
```

Reply:
do you want me to change it?

Reply:
just informational for now, but you might as well :)

Reply:
I had a quick look and it's not obvious to me how to make the change, if you could just add this to the cleanup backlog I'd really appreciate it. The new form tests look good!